src/components/SearchGrid.vue

```diff
@@ -5,19 +5,20 @@
       <div class="search-grid_analytics cell medium-6 large-6" >
         <h5>
           <span>{{ _imagesCount }}</span>
-          <span>{{ searchTerm }}</span>
-          Photos
+          photos matching
+          <span>'{{ searchTerm }}'</span>
         </h5>
       </div>
       <div class="search-grid_layout-control cell medium-6 large-6">
-        <h5>Grid Options:</h5>
+        <h5>Image options:</h5>
         <fieldset>
           <input
-            id="watermark"
+            id="scaling"
             type="checkbox"
+            v-on:change="handleScalingChange"
             v-model="shouldContainImages">
-            <label for="watermark">
-              Contain image
+            <label for="scaling">
+              Prevent overscaling
```

Comment:
This didn't make any changes in the layout in my test cases. How can I see this in action?

Reply:
This button works on inages that have very small thumbnails. Essentially in a masonry layout the width of each cell is constant, but if we set img width to 100% of the cell, small images get stretched. This option allows very small images to not be scaled up and rather appear with vertical grey bar on both sides, emphasising true size and no pixelation over consistent appearance.

Reply:
The fact that the checkbox had almost no visible effect is a testament to the effectiveness of the layout itself. However, the purpose of the checkbox is to handle images like this one:
![image](https://user-images.githubusercontent.com/16580576/54914694-8d02e800-4f1b-11e9-90dc-1f471f95af64.png)

This image has a particularly small width, less than even the width of a cell. This image would normally be scaled up (might introduce artefacts) and the checkbox prevents that from happening using vertical grey bars.